### ISSUE TITLE
the problem of evaluating model

### ISSUE THREAD

**Issue by huzheng1111** [SVAIGBA/WMSeg#7]
Excuse me,For the SIGHAN 2005 dataset, do you use the test data to evaluate the model, and then use the test data to test the final performance of the model?

**Comment by yuanheTian** [comment]
Thank you for your question.

We use a part of the training set as the dev set to find the best hyperparameter settings (e.g., batch size, learning rate, etc.) and then use that hyperparameter setting to train our model on the entire training set and finally evaluate it on the test set.
